app/Http/Controllers/PollController.php

```diff
@@ -17,7 +21,7 @@ class PollController extends Controller
      */
     public function index()
     {
-        return PollResource::collection(Poll::with('users', 'items')->paginate(25));
+        return new PollCollection(Poll::with('items')->paginate(25));
```

Comment:
Items are missing the "weight" attribute (sum of `rating`for this poll and item in the table `poll_ratings`)